src/modules/ekf2/EKF2.cpp

```diff
@@ -1362,6 +1345,28 @@ float EKF2::filter_altitude_ellipsoid(float amsl_hgt)
 	return amsl_hgt + _wgs84_hgt_offset;
 }
 
+void EKF2::UpdateAirspeedSample(ekf2_timestamps_s &ekf2_timestamps)
+{
+	// EKF airspeed sample
+	airspeed_s airspeed;
+
+	if (_airspeed_sub.update(&airspeed)) {
+		// only set airspeed data if condition for airspeed fusion are met
+		if ((_param_ekf2_arsp_thr.get() > FLT_EPSILON) && (airspeed.true_airspeed_m_s > _param_ekf2_arsp_thr.get())) {
+
+			airspeedSample airspeed_sample {
+				.true_airspeed = airspeed.true_airspeed_m_s,
+				.eas2tas = airspeed.true_airspeed_m_s / airspeed.indicated_airspeed_m_s,
+				.time_us = airspeed.timestamp,
+			};
```

Comment:
Is it better to fill the struct this way? You can then add a `const` I would say.

Reply:
Depending on the initialization specifics it can be slightly better, but nothing to get excited about. This is compact, discourages unnecessary initialization, and forces you to set every field.